### PR TITLE
Python: Dictionary_Comprehension #3200

### LINKED ISSUE #3200: Python: Dictionary_Comprehension
### Description

Description:
What is dictionary comprehension?
Why is dictionary comprehension is needed?
Utility of dictionary comprehension with examples
Some advantages

### Domain

Python_Domain

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by rex2828**
<hr>

## Description 📜

Added a video on Dictionary Comprehension.
Fixes #3200
<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [x] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [x] Python_Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

### CODE REVIEW

**Comment by nayakaj** [comment]
can u plz assign this to me

File: Python/Dictionary_Comprehension/dictionary_comprehension_video.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## Dictionary Comprehension Video
```
Comment (apeksha235) [inline Python/Dictionary_Comprehension/dictionary_comprehension_video.md:1]: @rex2828 Great work on the video!! 
- Just mention the topics covered in your md file
- Add the relative link of your file to the readme of the folder

**Review by apeksha235 — CHANGES_REQUESTED** [review]

File: Python/Dictionary_Comprehension/dictionary_comprehension_video.md (hunk 1 above)
Comment (rex2828) [reply Python/Dictionary_Comprehension/dictionary_comprehension_video.md:1]: @apeksha235 Dictionary Comprehension folder is already added in the Readme, do you want me to add the seperate link for my md file or what?

**Comment by kumarankm** [comment]
@apeksha235 Can we close this pr?

**Comment by apeksha235** [comment]
@kumarankm, i guess we can merge this. what do you think?

**Comment by kumarankm** [comment]
Yes, we can.
@adityabisoi you can merge this pr

**Pull request merged**